example-workflows/connector/.github/workflows/docs.yml

```
@@ -22,25 +27,31 @@ jobs:
         with:
           java-version: 11
 
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
       - uses: eskatos/gradle-command-action@v1
         name: Update Documentation
         with:
           arguments: prepareDocs
 
+      - name: Generate Connector Name
+        run: |
+          echo "$( ./gradlew printConnectorDocumentationName -q)"
```

Why specifically are you creating a subshell, running a command, and then just echoing its output?

What'd be the difference to this?
```suggestion
          ./gradlew printConnectorDocumentationName -q
```

---

That should work exactly the same way, yes.